.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: ci
+
+on: push
+
+env:
+  CONFIGURATION: Release
+
+jobs:
+
+  buid:
+    runs-on: windows-2022
+    timeout-minutes: 10
```

Comment:
I propose to leave default one rather than limiting us

Reply:
Fixed.